### DISCUSSION TITLE
TextField Bug

### DISCUSSION THREAD

**Discussion by MrDIDJoy** [flet-dev/flet#3206] [Q&A] [answered]
### Question

Hello, I want to share a bug that I discovered today while creating an application
When I inserted the link into the Textfield, something strange happened to it. It's like most of it was cut off from the bottom

Images:
![image](https://github.com/flet-dev/flet/assets/79699545/a36cb75a-5892-40ea-99f3-746b9e221596)
![image](https://github.com/flet-dev/flet/assets/79699545/96a8e06d-baba-4a4d-bc31-c2e295255cd8)

### Code sample

```python
def main(page):
    page.horizontal_alignment = ft.CrossAxisAlignment.CENTER
    page.vertical_alignment = ft.MainAxisAlignment.CENTER
    page.title = AppName
    page.theme_mode = "dark"
    page.window_title_bar_hidden = True
    page.window_title_bar_buttons_hidden = True
    page.window_min_height = 800
    page.window_min_width = 700
    page.window_max_height = 800
    page.window_max_width = 700
    page.window_height = 800
    page.window_width = 700
    page.window_center()

    TextField(label="First button link", disabled=True, keyboard_type=KeyboardType.URL, width=250, 
              height=40, text_size=15)

    page.add(TextField)
ft.app(target=main, assets_dir="assets", view=FLET_APP)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Bbalduzz** [comment]
I believe this behaviour is due to the "incompatible" `height` and `padding` set in the textfield, try changing the TextField's `content_padding` (https://flet.dev/docs/controls/textfield/#content_padding)

**Comment by ndonkoHenri** [comment, marked as answer]
https://python.plainenglish.io/how-to-resize-flets-textfield-control-without-the-content-going-invisible-6d268a4d96e1
